src/models/vector_db_metadata.py

```diff
@@ -10,19 +11,22 @@ class VectorDBMetadata(Base):
     vector_db_type = Column(Enum(VectorDBType))
     index_name = Column(String)
     environment = Column(String)
+    collection = Column(String)
 
     def serialize(self):
         return {
             'vector_db_type': self.vector_db_type.name if self.vector_db_type else None,
             'index_name': self.index_name,
             'environment': self.environment,
+            'collection': self.collection
         }
     
     @staticmethod
     def _from_request(request):
-        vector_db_metadata_dict = json.loads(request.form.get('VectorDBMetadata'))
+        vector_db_metadata_dict = defaultdict(lambda : "", json.loads(request.form.get('VectorDBMetadata')))
```

Comment:
what is motivating this change?

Reply:
This is my approach for making the values in the dictionary optional. Using  [defaultdict](https://docs.python.org/3/library/collections.html#collections.defaultdict) prevents [KeyError](https://docs.python.org/3/library/exceptions.html#KeyError) when the user doesn't provide the expected keys. Rather it uses the returned value from the Lambda function as the default value. So in this case if the user doesn't provide collections it would just be `""`.